IOMC/EventVertexGenerators/python/VtxSmearedRealistic25ns13p6TeVEOY2022Collision_cfi.py

```diff
@@ -0,0 +1,7 @@
+import FWCore.ParameterSet.Config as cms
+
+from IOMC.EventVertexGenerators.VtxSmearedParameters_cfi import Realistic25ns13p6TeVEOY2022CollisionVtxSmearingParameters,VtxSmearedCommon
+VtxSmeared = cms.EDProducer("BetafuncEvtVtxGenerator",
+    Realistic25ns13p6TeVEOY2022CollisionVtxSmearingParameters,
+    VtxSmearedCommon
+)
```

Comment:
picky - but there is no newline at the end of the file ;)

Reply:
You are right I missed it.
I will fix this in the next PR (when we include the reco-BS and the new workflows).

Reply:
@francescobrivio , usually we fix promptly such requests. I will sign this PR but do not understand urgency.

Reply:
@civanch thanks for signing this!

> usually we fix promptly such requests

When I saw Marco's comment the tests had already started, so I didn't want to stop and repeat them just for a cosmetic empty line

>  I will sign this PR but do not understand urgency.

The urgency is to get this in 13_0_0_pre4, so we can get the reco-BS (and the matrix workflows) in 13_0_0 and get a proper validation.